src/components/heading/index.tsx

```diff
@@ -0,0 +1,2 @@
+import { Heading } from '@chakra-ui/react';
+export default Heading;
```

Comment:
Up for discussion: What I had in mind is to create wrappers so we can not mess up the combinations

```
const H1: FC<Omit<Props, "as" | "textStyle">> ({_as, _textStyle, children, ...rest}) => {
  return  
   <Heading
      as: 'h6',
      textStyle: 'heading6',
     {...rest}
     >
     {children}
     </Heading>
}
```

Reply:
Thank you for the review Lorenzo! 
It looks "safer", I will adjust it 👍